orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorJenkinsJobTask.groovy

```diff
@@ -61,7 +61,8 @@ class MonitorJenkinsJobTask implements OverridableTimeoutRetryableTask {
 
     def buildNumber = (int) stage.context.buildNumber
     try {
-      Map<String, Object> build = buildService.getBuild(buildNumber, master, job)
+      def user = stage.execution.authentication?.user
```

Comment:
This should be available on the `ExecutionContext` thread local (and potentially more likely to handle the scenario of their being a manual judgment w/ propagate auth in the middle of the pipeline)